src/interfaces/video.ts

```diff
@@ -0,0 +1,11 @@
+import { VideoType } from '../types/video-type';
+
+export interface Video {
+    description: string;
+    mediaType: VideoType;
+    publishDate: string;
+    thumbnailUrl: string;
+    title: string;
+    videoId: string;
+    playlistId?: string;
```

Comment:
We may not even need this field, I checked the repo and did not see it used anywhere. Do you think we can remove this one?

Reply:
Nice catch, I will remove this property here and from classes but let keep for response interfaces.